imap_processing/tests/conftest.py

```diff
@@ -90,13 +90,27 @@ def _download_de440s(spice_test_data_path):
 def pytest_collection_modifyitems(items):
     """
     The use of this hook allows modification of test `Items` after tests have
-    been collected. In this case, it automatically adds the _download_de440s
-    fixture to any test marked with the `external_kernel`.
+    been collected. In this case, it automatically adds fixtures based on the
+    following table:
+
+    +---------------------+---------------------+
+    | pytest mark         | fixture added       |
+    +=====================+=====================+
+    | external_kernel     | _download_de440s    |
+    | use_test_metakernel | use_test_metakernel |
+    +---------------------+---------------------+
+
+    Notes
+    -----
+    See the following link for details about this function, also known as a
+    pytest hook:
     https://docs.pytest.org/en/stable/reference/reference.html#pytest.hookspec.pytest_collection_modifyitems
     """
     for item in items:
         if item.get_closest_marker("external_kernel") is not None:
             item.fixturenames.append("_download_de440s")
+        if item.get_closest_marker("use_test_metakernel") is not None:
```

Comment:
This simplifies the use of the `use_test_metakernel` fixture my making the name and the `@pytest.mark` the same and automatically adding the fixture to any test marked as such.

Reply:
Oh that's pretty nice.